crates/core/src/auth/identity.rs

```diff
@@ -20,6 +20,26 @@ pub struct SpacetimeIdentityClaims {
     pub exp: Option<SystemTime>,
 }
 
+// The new token format that we are sending out.
```

Comment:
Is this related to this patch?

Reply:
The diff appears to be showing changes that were merged in https://github.com/clockworklabs/SpacetimeDB/pull/1854.